source/MRMesh/MRSubfeatures.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "MRMesh/MRFeatures.h"
+#include "MRMesh/MRFlagOperators.h"
+
+#include <functional>
+
+namespace MR
+{
+
+enum class OfferedSubfeatureFlags
+{
+    // This subfeature can also be reached indirectly through a different subfeature.
+    // It doesn't need to be displayed in the GUI to save space, but you can still display a 3D object for it for convenience.
+    indirect = 1 << 0,
+
+    // Visualizing this subfeature in 3D would be intrusive. Don't create a preview object by default.
+    visualizationIsIntrusive = 1 << 1,
```

Comment:
dont understand what is it for

Reply:
why is functions in MRMesh think about rendering

Reply:
Note that I'm just copying this from `MeasureFeaturesPlugin`. This disables features that look ugly in 3D, such as infinite lines or anything with a mesh.